src/api/app/models/bs_request_permission_check.rb

```diff
@@ -179,7 +179,7 @@ def check_accepted_action(action)
       begin
         Backend::Api::Sources::Package.files(action.source_project, action.source_package, query)
       rescue Backend::Error
-        raise ExpandError, "The source of package #{action.source_project}/#{action.source_package}#{action.source_rev ? " for revision #{action.source_rev}" : ''} is broken"
+        raise ExpandError, "The source of package #{action.source_project}/#{action.source_package}#{" for revision #{action.source_rev}" if action.source_rev} is broken"
```

Comment:
I would even move the condition outside the string and build two different strings. I think putting conditions inside string interpolation is too much...

Reply:
Sure. I added the changes in other commit. Please take a look again.